### ISSUE TITLE
AutoSizer width: 0 issue 

### ISSUE THREAD

**Issue by sontek** [bvaughn/react-virtualized#634]
I've read the FAQ/docs on AutoSizer and it says it fills the height/width of the container, so I basically have this snippet of code to try to use autosizer:

```
import { AutoSizer } from 'react-virtualized';
import React from 'react';
import createHelper from 'recompose/createHelper'; // Helper for the dev mode.

export default createHelper(
    (options) => (Component) => {
        const Dimensions = props => (
            <div style={{width: "100%", "background-color": "#FF0000"}}>
                <AutoSizer {...options} disableHeight>
                    {newProps => (
                        <Component
                            containerWidth={newProps.width}
                            {...props}
                        />
                    )}
                </AutoSizer>
            </div>
        );
        return Dimensions;
    },
    "dimensions"
);
```

which basically puts a container with 100% width around the autosizer but I still get `width: 0`:

<img width="1446" alt="screenshot 2017-03-29 14 04 21" src="https://cloud.githubusercontent.com/assets/151924/24471796/e287c478-1488-11e7-9429-570c2388b2d3.png">

You can see the container has a red background, so it is filling 100% of the container width, but the autoSizer is still getting 0.

**Comment by bvaughn** [comment]
Are you sure you aren't seeing the red background from the element above the one you have selected? Or that `props` doesn't override `containerWidth`? Or that `containerWidth` is actually being passed through and set as `style={{ width: containerWidth}}`? There are a lot of missing pieces.

I copied your code above (but removing references to code you didn't share) here: https://plnkr.co/edit/ggxVDaLEgQY0Jdnkz2qm?p=preview

Works fine for me. Closing this issue since I assume the error is in your app code somewhere. We can keep chatting here though. 😄
